js/components/AboutPage.jsx

```diff
@@ -3,28 +3,47 @@
 var Footer = require('./Footer.jsx');
 var Toolbar = require('./Toolbar.jsx');
 var React = require('react');
+var strings = require('../lib/strings_utils.js');
 
 var AboutPage = React.createClass({
   render: function() {
     return (
       <div className="aboutPage">
-        <Toolbar />
-        <div className="aboutContent container-fluid">
-          <h2>About Repcoin</h2>
-          <div className="aboutReps">
-            <p>Repcoin is the first market-based approach to reputation. Use it to discover talent and prove yourself in your field.</p>
-
-            <p>Repcoin hosts a centralized, unbiased source of reputation for experts and investors on countless topics.</p>
-            <p>Sign up as an expert and showcase your talent for a given skill. Sign up as an investor and prove your eye for the best in the field.</p>
-
-            <p>Repcoin ranks experts and investors within a given category on a bell curve. Your expert percentile is determined by the investments you receive, and your investor percentile is derived from the return on your investments.</p>
-
-            <p>Your Repcoin reputation is portable. Display it on third party sites and use it as a pure rating for your skill.</p>
-
-            <p>Choose your categories and start contributing to the purest metric of success on the web!</p>
+        <div className="row">
+          <Toolbar />
+        </div>
+        <div className="row">
+          <div className="col-md-8 col-md-offset-1 aboutContent">
+            <h1>Repcoin FAQ</h1>
+            <br/>
+            <h2>What is Repcoin?</h2>
+            <p>Repcoin is a market for reputation. As you turn to the internet more and more for entertainment and information, it becomes harder to find good content. And, for those putting content online, from artists to athletes, it becomes harder to get found. Repcoin is a discovery platform. Users can sign up as an investor or expert for a self-proclaimed skill. Investors give reps to experts they believe in, and they can make huge returns from those investments. This helps your choice experts trend online and get found. It also boosts the investor's credibility for finding talent. Repcoin's data creates reliable and universal reputation. It can be shared all over the web, so that your comments anywhere can be annotated with your Repcoin score.</p>
```

Comment:
Only comment is that I'd appreciate some formatting on these. It's difficult to read them when their line length is  >>>100.